app/public/manifest.json

```diff
@@ -1,21 +1,21 @@
 {
-  "short_name": "Ionic App",
-  "name": "My Ionic App",
+  "short_name": "AdamRMS",
+  "name": "AdamRMS App",
   "icons": [
     {
-      "src": "assets/icon/favicon.png",
-      "sizes": "64x64 32x32 24x24 16x16",
+      "src": "https://cdn.adam-rms.com/img/icons/favicon-96x96.png",
```

Comment:
Is this how manifest is generated?

Reply:
This was based off the original file so I'm assuming so?